### PR TITLE
added inline disconnect button for chromecast

### PR DESCRIPTION
**Pull request by vinodf2f**

### CODE REVIEW

File: renderer/components/Container/Container.tsx [hunk 1]
```diff
@@ -152,7 +157,15 @@ export default function () {
     >
       <Header onFileSelect={onFileSelect} />
       <Message show={!!selectedCast?.host}>
-        You are connected to {selectedCast?.name}
+        <div className={styles.castMessageContainer}>
+          You are connected to {selectedCast?.name}
+          <text
```
Comment (ritz078) [inline renderer/components/Container/Container.tsx:162]: I haven't seen `text` tag ever ? Is this documented somewhere ? I couldn't find it online too.

File: renderer/components/Container/Container.tsx [hunk 2]
```diff
@@ -31,7 +34,9 @@ export default function () {
   );
 
   const [color, setColor] = useState<string>(store.get("color") as string);
-  const { selectedCast } = useContext(SelectedCastContext);
+  const { selectedCast, setSelectedCast } = useContext(
+    SelectedCastContext
+  ) as castContext;
```
Comment (ritz078) [inline renderer/components/Container/Container.tsx:39]: was adding `as castContext` needed ? It should automatically detect this.

**Review by ritz078 — COMMENTED** [review]
Great job. I have left few comments for your consideration.

File: renderer/components/Container/Container.tsx (hunk 1 above)
Comment (vinodf2f) [reply renderer/components/Container/Container.tsx:162]: Apologies. Coming from React-Native, I added this by mistake. will convert it to `p` tag.

File: renderer/components/Container/Container.tsx (hunk 2 above)
Comment (vinodf2f) [reply renderer/components/Container/Container.tsx:39]: Yes, I am not sure why but it is not detecting.

**Review by ritz078 — APPROVED** [review]
Great work. LGTM.

**Comment by ritz078** [comment]
@all-contributors please add @vinodf2f for code.

**Pull request merged**